src/api/coderApi.ts

```diff
@@ -457,7 +507,7 @@ function addLoggingInterceptors(client: AxiosInstance, logger: Logger) {
 		},
 		(error: unknown) => {
 			logError(logger, error, getLogLevel());
-			return Promise.reject(error);
+			throw error;
```

Comment:
Just to double check, the docs I saw do use `Promise.reject()`, are we guaranteed `throw` works as well?